src/security.md

```diff
@@ -7,17 +7,63 @@ created with it seriously.
 This page describes how to report any vulnerabilities that you find,
 and lists best practices to minimize the risk of introducing a vulnerability.
 
+## Security philosophy
+
+Dart's security strategy is based on five key pillars:
+
+* **Identify**: Track & prioritize key security risks by identifying core assets, 
+key threats and vulnerabilities. 
+* **Detect**: Detect and identify vulnerabilities using techniques and tools like
+ vulnerability scanning,
+  static application security testing and fuzzing.
+* **Protect**: Eliminate risks by mitigating known vulnerabilities and protect 
+critical assets against source threats.
+* **Respond**: Define processes to report, triage and respond to vulnerabilities 
+or attacks.
+* **Recover**: Build capabilities to contain and recover from an incident with 
+minimal impact.
```

Comment:
Suggestions to fix the whitespace inconsistencies @mit-mit mentioned and stay in our recommended width with semantic breaks.

```suggestion
* **Identify**: Track & prioritize key security risks by identifying 
  core assets, key threats and vulnerabilities.
* **Detect**: Detect and identify vulnerabilities using techniques and tools
  like vulnerability scanning, static application security testing, and fuzzing.
* **Protect**: Eliminate risks by mitigating known vulnerabilities and
  protect critical assets against source threats.
* **Respond**: Define processes to report, triage, and
  respond to vulnerabilities or attacks.
* **Recover**: Build capabilities to contain and recover from an incident
  with minimal impact.
```